### PR TITLE
Integrating the fmp submenu into the fa menu + plotting + ratios

### PR DESCRIPTION
**Pull request by northern-64bit**
# Description

PR moved from #2060, adressing #1866, due to not working tests

# How has this been tested?

Useing the commands

# Checklist:

- [x] Update [our Hugo documentation](https://openbb-finance.github.io/OpenBBTerminal/) following [these guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/tree/main/website).
- [x] Update our tests following [these guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/tree/main/tests).
- [x] Make sure you are following our [CONTRIBUTING guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/blob/main/CONTRIBUTING.md).
- [x] If a feature was added make sure to add it to the corresponding [scripts file](https://github.com/OpenBB-finance/OpenBBTerminal/tree/main/scripts).

# Others
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My code passes all the checks pylint, flake8, black, ... To speed up development you should run `pre-commit install`.
- [x] New and existing unit tests pass locally with my changes. You can test this locally using `pytest tests/...`.

### CODE REVIEW

**Comment by Chavithra** [comment]
Hi @northern-64bit,

I have updated the tests you can look at the commit to see how I fixed those.

Thanks

File: i18n/en.yml [hunk 1]
```diff
@@ -364,8 +364,13 @@ en:
   stocks/fa/earnings: earnings dates and reported EPS
   stocks/fa/fraud: key fraud ratios
   stocks/fa/dupont: detailed breakdown for return on equity
+  stocks/fa/profile: company profile
+  stocks/fa/quote: detailed stock quote information
+  stocks/fa/enterprise: company enterprise value
+  stocks/fa/metrics: key metrics over time
+  stocks/fa/ratios: in-depth ratios over time
+  stocks/fa/growth: growth of financial statement items and ratios
   stocks/fa/_sources_: Other sources
```
Comment (JerBouma) [inline i18n/en.yml:373]: Remove as we have no other sources.

File: i18n/en.yml [hunk 2]
```diff
@@ -364,8 +364,13 @@ en:
   stocks/fa/earnings: earnings dates and reported EPS
   stocks/fa/fraud: key fraud ratios
   stocks/fa/dupont: detailed breakdown for return on equity
+  stocks/fa/profile: company profile
+  stocks/fa/quote: detailed stock quote information
+  stocks/fa/enterprise: company enterprise value
```
Comment (JerBouma) [inline i18n/en.yml:369]: Add in the `dcfc` command. The text can be "determine the (historical) discounted cash flow"

**Review by JerBouma — COMMENTED** [review]
The following commands are failing (with TSLA loaded):

```
2022 Jul 19, 04:07 (🦋) /stocks/fa/ $ score
/Users/jeroenbouma/opt/anaconda3/envs/gst/lib/python3.8/site-packages/pandas/core/indexes/base.py:6982: FutureWarning: In a future version, the Index constructor will not infer numeric dtypes when passed object-dtype sequences (matching Series behavior)
  return Index(sequences[0], name=names)
/Users/jeroenbouma/opt/anaconda3/envs/gst/lib/python3.8/site-packages/pandas/core/indexes/base.py:6982: FutureWarning: In a future version, the Index constructor will not infer numeric dtypes when passed object-dtype sequences (matching Series behavior)
  return Index(sequences[0], name=names)
/Users/jeroenbouma/opt/anaconda3/envs/gst/lib/python3.8/site-packages/pandas/core/indexes/base.py:6982: FutureWarning: In a future version, the Index constructor will not infer numeric dtypes when passed object-dtype sequences (matching Series behavior)
  return Index(sequences[0], name=names)
/Users/jeroenbouma/opt/anaconda3/envs/gst/lib/python3.8/site-packages/pandas/core/indexes/base.py:6982: FutureWarning: In a future version, the Index constructor will not infer numeric dtypes when passed object-dtype sequences (matching Series behavior)
  return Index(sequences[0], name=names)
Score: 32.22 %

2022 Jul 19, 04:07 (🦋) /stocks/fa/ $ dcfc
Error: Length of headers does not match length of DataFrame
```

File: i18n/en.yml (hunk 1 above)
Comment (JerBouma) [reply i18n/en.yml:373]: Also remove this from the fa controller.

**Comment by northern-64bit** [comment]
Everything except for the score command was fixed, since the error was in the valinvest library. Should I open a issue in its github and maybe try to fix it with a pr? The library hasn't been updated since September 2020 so maybe it isn't optimal to have it as a dependency.

**Comment by JerBouma** [comment]
Perhaps you could suppress that warning? E.g. https://stackoverflow.com/questions/14463277/how-to-disable-python-warnings

**Comment by northern-64bit** [comment]
> Perhaps you could suppress that warning. Just make to re-enable after. E.g. https://stackoverflow.com/questions/14463277/how-to-disable-python-warnings

Fixed!

**Review by JerBouma — APPROVED** [review]
Looks good!

**Pull request merged**